ui/raidboss/emulator/overrides/RaidEmulatorTimelineController.js

```diff
@@ -43,11 +43,10 @@ export default class RaidEmulatorTimelineController extends TimelineController {
     e.detail.logs.forEach((line) => {
       this.activeTimeline.emulatedTimeOffset = line.offset;
       this.ui.emulatedTimeOffset = line.offset;
-      this.activeTimeline.timebase = this.activeTimeline.timebase || line.timestamp;
       this.activeTimeline.OnLogLine(
           line.properCaseConvertedLine || line.convertedLine,
           line.timestamp);
-      this.activeTimeline._OnUpdateTimer(line.timestamp);
+      this.activeTimeline.timebase && this.activeTimeline._OnUpdateTimer(line.timestamp);
```

Comment:
I'm avoiding calling it because if `_OnUpdateTimer` is called without a `timebase` value set, it throws a console log, and 4000-5000 console logs per encounter analysis is a bit excessive.

Reply:
Changed to this for the next commit:

```js
      // Only call _OnUpdateTimer if we have a timebase from the previous call to OnLogLine
      // This avoids spamming the console with a ton of messages
      if (this.activeTimeline.timebase)
        this.activeTimeline._OnUpdateTimer(line.timestamp);
```